feature-group-definitions/css-comparison.yml

```diff
@@ -0,0 +1,6 @@
+name: CSS min(), max(), and clamp()
```

Comment:
Maybe the ID is off too? `comparison-functions`?

Reply:
Maybe just min-max-clamp since that's even shorter?